src/styles/utilits/_mixins.scss

```
@@ -0,0 +1,11 @@
+
+@mixin image-square($size) {
```

```suggestion
@mixin square($size) {
```

Can i use this mixin with div?  Or its only for img?